### PR TITLE
Full-width buttons on mobile

### PR DESCRIPTION
**Pull request by iamkeir**
Full width buttons on mobile (< 420px) for RP.

@Jonic not sure if we want this to be a `primitive` rather than just RP?

### CODE REVIEW

File: src/components/raspberry-pi/button/_index.scss [hunk 1]
```diff
@@ -19,6 +20,10 @@
   transition-property: background-color, transform;
   transition-timing-function: transition-timing.$eased;
 
+  @include device.mq('max-width', 420px) {
+    @include size.width(100%);
```
Comment (iamkeir) [inline src/components/raspberry-pi/button/_index.scss:24]: @Jonic not sure how you feel about `max-width` media queries but can switch to `min-width` if we prefer.

Comment (Jonic) [reply src/components/raspberry-pi/button/_index.scss:24]: @iamkeir Yeah as a rule of thumb we should avoid `max-width` where possible. It looks innocent enough here, but I don't want people to see `max-width` and think that it's a thing that is good and fine to use. If you could swap this out then that would be SUPER :)

**Review by Jonic — CHANGES_REQUESTED** [review]
Just the min-width thing to update and this is aaaaall good :)

**Review by Jonic — APPROVED** [review]
LGTM

**Pull request merged**